cmd/notary/tuf.go

```diff
@@ -503,15 +481,16 @@ func bootstrapRepo(gun string, repo *tuf.TufRepo) store.MetadataStore {
 		fatalf(err.Error())
 	}
 	repo.SetSnapshot(snapshot)
-	timestampJSON, err := filestore.GetMeta("timestamp", 0)
-	if err != nil {
-		fatalf(err.Error())
-	}
-	timestamp := &data.Signed{}
-	err = json.Unmarshal(timestampJSON, timestamp)
-	if err != nil {
-		fatalf(err.Error())
-	}
-	repo.SetTimestamp(timestamp)
 	return filestore
 }
+
+// Use this to initialize remote HTTPStores from the config settings
+func getRemoteStore(gun string) (store.RemoteStore, error) {
```

Comment:
This should probably take a transport and baseURL instead of assuming HTTP and going to notary:4443